src/PositionManager.sol

```diff
@@ -50,6 +50,8 @@ contract PositionManager is
 
     IAllowanceTransfer public immutable permit2;
 
+    uint256 public constant FULL_DELTA = type(uint256).max;
```

Comment:
I do think youll find if you make this `0` gas should go down noticeably because of removing non-0 calldata bytes.
It _should_ go down by `(16-4) * 32) = 384`

Reply:
lemme check!

Reply:
went down by ~700

Reply:
https://github.com/Uniswap/v4-periphery/pull/218/commits/b0b1b7321e55e689f0b24262f746db6c02e424c1